bundles/org.openhab.binding.hydrawise/src/main/java/org/openhab/binding/hydrawise/internal/handler/HydrawiseControllerHandler.java

```diff
@@ -12,7 +12,37 @@
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_CONTROLLER_SUMMARY;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_CONDITIONS;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_EVAPOTRANSPRIATION;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_HUMIDITY;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_PRECIPITATION;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_PROBABILITYOFPRECIPITATION;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_TEMPERATURE_HIGH;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_TEMPERATURE_LOW;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_TIME;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_FORECAST_WIND;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_GROUP_ALLZONES;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_GROUP_CONTROLLER_SYSTEM;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_SENSOR_ACTIVE;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_SENSOR_DELAY;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_SENSOR_NAME;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_SENSOR_OFFLEVEL;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_SENSOR_OFFTIMER;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_SENSOR_WATERFLOW;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_DURATION;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_ICON;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_NAME;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_NEXT_RUN_TIME_TIME;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_RUN;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_RUN_CUSTOM;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_SUMMARY;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_SUSPEND;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_SUSPENDUNTIL;
+import static org.openhab.binding.hydrawise.internal.HydrawiseBindingConstants.CHANNEL_ZONE_TIME_LEFT;
```

Comment:
Minor, but wildcard imports are allowed for static's
This can be reverted

Reply:
yeah , thats odd.  I have been using Visual Studio code lately for development, i have a rule which runs:

`mvn spotless:apply -DspotlessFiles=${file}` 

On every file save, but spotless seems to be auto expanding my import!

Reply:
actually i think something else must be doing it in the IDE, in any case, i'll fix it.